questiontype.php

```diff
@@ -398,6 +398,13 @@ public function get_available_questions_with_tags($difficulty, $topic, $category
         ';
 
         $questionids = $DB->get_record_sql($query);
+        // If no question with specified data is found, die
+        // TODO -> remove the question last introduced, since the same error is thrown
+        //      every time the user tries to acces/edit the quiz
+        if (!$questionids) {
+            echo ' <script> alert("No questions found"); </script> ';
```

Comment:
Be more specific in the message:
e.g. "No questions found having the selected difficulty and topic"